templates/default/fulldoc/html/js/app.js

```diff
@@ -120,6 +120,38 @@ function summaryToggle() {
   } else { localStorage.summaryCollapsed = "expand"; }
 }
 
+function summaryToggleConst() {
+  $('.summary_const_toggle').click(function(e) {
+    e.preventDefault();
+    localStorage.summaryCollapsed = $(this).text();
+    $('.summary_const_toggle').each(function() {
+      $(this).text($(this).text() == "collapse" ? "expand" : "collapse");
+      var next = $(this).parent().parent().nextAll('dl.summary').first();
+      if (next.hasClass('compact')) {
+        next.toggle();
+        next.nextAll('dl.summary').first().toggle();
+      }
+      else if (next.hasClass('summary')) {
+        var list = $('<dl class="constants summary compact" />');
+        list.html(next.html());
+        list.find('.docstring').remove();
```

Comment:
Why are individual elements being removed here (including elements below). Why not just use CSS to style this?